recipes/cuda-profiler-api/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "cuda-profiler-api" %}
+{% set version = "12.0.76" %}
+{% set cuda_version = "12.0" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-ppc64le" %}  # [ppc64le]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set platform = "windows-x86_64" %}  # [win]
+{% set target_name = "x86_64-linux" %}  # [linux64]
+{% set target_name = "ppc64le-linux" %}  # [ppc64le]
+{% set target_name = "sbsa-linux" %}  # [aarch64]
+{% set target_name = "x64" %}  # [win]
+{% set extension = "tar.xz" %}  # [not win]
+{% set extension = "zip" %}  # [win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/cuda_profiler_api/{{ platform }}/cuda_profiler_api-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: 0fb470a3065ad39666dd231c12f8c8a9a94f34c4ac8494e5a399ea8e6ad69346  # [linux64]
+  sha256: 211513a8a194557a2b838a46700a820af6f38932fac5a82fa229c162779d9a25  # [ppc64le]
+  sha256: b93d4e99077c31e7034eeaea09068b43abc0dd2868a01d489ebcd1e51ceab816  # [aarch64]
+  sha256: e8bccbfda35abc7076ba6623578d11d05966a9c7fed817cbfe7d4dea9627057e  # [win]
+
+build:
+  number: 0
+  skip: true  # [osx]
+
+requirements:
+  build:
+    - arm-variant * {{ arm_variant_type }}  # [aarch64]
+  host:
+    - cuda-version {{ cuda_version }}
+  run:
+    - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+    - cuda-cudart-dev
+  run_constrained:
+    - arm-variant * {{ arm_variant_type }}  # [aarch64]
+
+test:
+  commands:
+    - test -f $PREFIX/targets/{{ target_name }}/include/cuda_profiler_api.h  # [linux]
+    - if not exist %LIBRARY_INC%\cuda_profiler_api.h exit 1  # [win]
```

For good measure, let's check for the headers with the driver API too:
```suggestion
    - test -f $PREFIX/targets/{{ target_name }}/include/cuda_profiler_api.h  # [linux]
    - test -f $PREFIX/targets/{{ target_name }}/include/cudaProfiler.h       # [linux]
    - if not exist %LIBRARY_INC%\cuda_profiler_api.h exit 1  # [win]
    - if not exist %LIBRARY_INC%\cudaProfiler.h exit 1       # [win]
```